### PR TITLE
Log should be able to handle an Error Object

### PR DESCRIPTION
**Pull request by switters**
Dev should be able to directly hand the logger an error.

Currently, a passed error becomes an empty object in message.

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -43,6 +43,17 @@ test('send calls onsend', function(){
   lo.send('info', 'test', {foo:true})
 })
 
+test('can send error objs', function () {
+  function spy(msg) {
+    assert(msg.message.message === errMsg)
```
Comment (lvivier) [inline test/index.js]: lol "Jasper…Jasper…Jasper? which message property has the message?"

**Comment by lvivier** [comment]
:+1: looks great

File: test/index.js (hunk 1 above)
Comment (switters) [inline test/index.js]: :beers: DDC in a PR comment... never in my wildest dreams.

**Comment by switters** [comment]
and now we have stack traces when Errors are passed to the logger...

screen shot 2015-02-04 at 4 49 47 pm

**Comment by switters** [comment]
'wrong message' test is still failing. Will remedy in correlated issue #16. Going up! :beers:

**Pull request merged**